Assets/Scripts/AStar/Path.cs

```diff
@@ -0,0 +1,18 @@
+﻿using UnityEngine;
+using System.Collections;
+
+namespace AStar
+{
+    public class Path
```

Comment:
Consider using `struct` instead of `class` here since Path is a data store with no functionality.